src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import { useNotesContext } from "../../utils/contexts/NotesContext";
+import ColorOptions from "../ColorOptions";
+import "./Modal.styles.css";
+function Modal(props) {
+  const { note, closeEditing } = props;
+  const { notes, saveNotes } = useNotesContext();
+  const [values, setValues] = useState(note);
+  const [openMenu, setOpenMenu] = useState(false);
+
+
+  const { title, text, color } = values;
+
+  const editNote = () =>{
+    const newList = [...notes];
+    const edited = newList.find((f) => f.id === note.id);
+    edited.title = values.title;
+    edited.text = values.text;
+    edited.color = values.color;
+    saveNotes(newList);
+    closeEditing();
+  };
+
+  const changeColor = (color) => {
+    if (color === values.color) setValues({ ...values, color: "default" });
```

Comment:
when working with react, single quotes are used for string definition, and in the HTML markup the properties use double quotes